servers/pollux/default.nix

```diff
@@ -0,0 +1,163 @@
+{ modulesPath, inputs, config, pkgs, ... }:
+let
+  vs = config.vault-secrets.secrets;
+
+  suitecrm = import ./suitecrm.nix { inherit pkgs; system = pkgs.system; };
+in {
+  imports = [
+    inputs.serokell-nix.nixosModules.ec2
+    inputs.hermetic.nixosModules.hermetic
+  ];
+
+  networking.firewall.allowedTCPPorts = [ 80 443 ];
+
+  users.users.suitecrm = {
+    isNormalUser = true;
+    home = "/var/www/suitecrm";
+    uid = 1024;
+    group = "suitecrm";
+  };
+
+  users.groups.suitecrm = {
+    gid = 1024;
+  };
+
+  systemd.services.nginx.preStart = pkgs.lib.mkForce "";
+
+  systemd.services.suitecrm =
+    { before = [ "phpfpm-suitecrm.service" ];
+      wantedBy = [ "multiuser.target" ];
+      script =
+        ''
+          set -xe
+
+          _www_dir="/var/www/suitecrm"
+          _base_dir="/var/suitecrm"
+          _persist_folders=("custom" "themes" "data" "upload")
+
+          if ! [[ -f "$_www_dir/.first_run" ]]
+          then
+            touch "$_www_dir/.first_run"
+
+            mkdir -p "$_www_dir"
+            cp -Raf "${suitecrm}/." "$_www_dir/"
+
+            mkdir -p $_base_dir
+            for dir in ''${_persist_folders[@]}
+            do
+                mkdir -p "$_base_dir/$dir"
+                cp -Raf "${suitecrm}/$dir/." "$_base_dir/$dir/"
+                rm -r "$_www_dir/$dir"
+                ln -s "$_base_dir/$dir" "$_www_dir/$dir"
+            done
+
+            touch "$_base_dir/config.php"
+            ln -s "$_base_dir/config.php" "$_www_dir/config.php"
+          fi
+
+          chmod +rx /var/www
+          find /var/www/suitecrm -type f -exec chmod 640 {} \;
+          find /var/www/suitecrm -type d -exec chmod 750 {} \;
+          chown suitecrm:nginx -R "$_www_dir"
+          chown suitecrm:nginx -R "$_base_dir"
+        '';
+
+      serviceConfig.Type = "oneshot";
+    };
+
+  services.mysql = {
+    enable = true;
+
+    ensureUsers =
+      [ { name = "suitecrm";
+          ensurePermissions = {
+            "suitecrm.*" = "ALL PRIVILEGES";
+          };
+        }
+      ];
+
+    ensureDatabases =
+      [ "suitecrm"
+      ];
+  };
+
+  services.nginx = {
+    enable = true;
+    recommendedProxySettings = true;
```

Comment:
I think `recommendedProxySettings` is already added by `serokell.nix`

Reply:
noted